package.json

```diff
@@ -26,5 +24,18 @@
   "bugs": {
     "url": "https://github.com/secondstreet/talker.js/issues"
   },
-  "homepage": "https://github.com/secondstreet/talker.js"
+  "homepage": "https://github.com/secondstreet/talker.js",
+  "devDependencies": {
+    "prettier": "^1.15.2",
+    "ts-loader": "^5.3.1",
+    "tslint": "5.7.0",
+    "tslint-config-prettier": "1.9.0",
+    "tslint-plugin-prettier": "^2.0.1",
```

Comment:
I think you still need to actually use this in the tslint.json file.

Reply:
Or if you don't want to run prettier through eslint, just remove it.

Reply:
After thinking about it more, I think I'd rather run prettier through a pre-commit hook like we do the rest of our repos.

Reply:
Does eslint not run on pre-commit?

Reply:
It runs as part of `yarn test`, which CircleCI is now configured to run on PRs.

Reply:
I'd highly recommend at some point switching all our repos to use `eslint-plugin-prettier`/`tslint-plugin-prettier` and run `eslint --fix` instead of `prettier --write` on pre-commit; no reason to let lint issues be committed and then caught in the PR..

Reply:
Yeah the biggest reason we haven't switched over right now is because eslint runs via Ember CLI's test runner, rather than as a standalone process. But that's [going away](https://github.com/ember-cli/rfcs/pull/121) soon.